Sources/UI/Views/ImagePollCell.swift

```diff
@@ -0,0 +1,368 @@
+//
+//  ImagePollCell.swift
```

Comment:
Same comments as `TextPollCell`. Most notably, consider using `UIStackView` to simplify layout logic.